### DISCUSSION TITLE
How to do image resizing & compressing

### DISCUSSION THREAD

**Discussion by vipafattal** [nhost/nhost#1602] [Q&A]
I'm not sure to ask my question here, but I am using Flutter Nhost SDK with Storage to upload public images, however, I need to resize images and compress them before uploading. 

I don't see a baked option with SDK so is best practice to use Cloud Function with middleware for resizing & compressing and use this function from the front-end or do that from the front-end. Looking for your recommendation, thanks

**Comment by suplere** [comment]
Hello,

I use this library for image operation before storing images to the Storage.
https://github.com/mat-sz/imtool
